sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -140,6 +141,55 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("check outputs of expression examples") {
```

Comment:
The fixes are good. How long does this take to run, BTW? just want to make sure it's not huge to rerun this every time, though I agree testing examples is useful.

Reply:
~ 15 seconds on my laptop

Reply:
Could we do it in parallel?

Reply:
I will do that.

Reply:
Running the test in parallel takes ~5-6 seconds on my laptop now.